src/api/estados.js

```diff
@@ -150,7 +150,10 @@ api.getAllRegions = ({ shouldReturnEntireJson = false }: {shouldReturnEntireJson
   return map(regions, region => region.regionName);
 };
 
-api.getStateRegion = ({ state = requiredParam('state') }: { state: string }): regionType => {
+api.getStateRegion = ({ state = requiredParam('state') }: { state: string }): regionType | {} => {
+  if (typeof state !== `string`) {
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)